src/annotations/api/index.ts

```diff
@@ -22,7 +22,17 @@ import {formatAnnotationQueryString} from 'src/annotations/utils/formatQueryStri
 export const writeAnnotation = async (
   annotations: Annotation[]
 ): Promise<Annotation[]> => {
-  const res = await axios.post(url, annotations)
+  // we need to convert the annotation object's start and end time fields to be
+  // string types so they can be parsed in the backend.
+  const annotationsRequestConverted = annotations.map(annotation => {
+    return {
+      ...annotation,
+      start: new Date(annotation.start).toISOString(),
```

Comment:
why is this being changed to strings here?

and the naming is still not consistent (start/end vs different names on the annotations... (startTime and stopTime?  don't have time to look them up here)

also:  how is readAnnotation affected?

Reply:
That is because according to the issue we'd like to handle the conversions at the boundary layer. Any suggestions for a better place we could do it?

I can rename the variables. 

Do you mean `getAnnotation` by chance? It is not touched in this PR.

Reply:
why are we doing the conversions at all? this is a new service, it shouldn't need conversions @hoorayimhelping

Reply:
re the naming. annotations service is expecting `start` and `end`:
```go
	EndTime   *time.Time        `json:"end,omitempty"`      // EndTime is the time of the event being annotated. Defaults to now if not set.
	StartTime *time.Time        `json:"start,omitempty"`    // StartTime is the start time of the event being annotated. Defaults to EndTime if not set.
}
```

The important bits for communicating are in the json tag (`json:"start"`). That defines what is expected in the json serialized body. I'm not sure if this renaming of vars from `start`->`startValue` is going to result in a payload with `"startValue": "some time", "endValue": "another time"}`, but that would be incorrect.

re the conversions. the expected time format is [`RFC3339Nano`](https://golang.org/pkg/time/) i can't say whether converting them a string is right or wrong, as long as it serializes as that format.